CHANGELOG.md

```diff
@@ -4,6 +4,86 @@ All notable changes to this project will be documented in this file.
 
 <!-- New entries go after this line -->
 
+## v3.3.0 - 2019-10-??
+
+For more details, [see the full changelog](https://github.com/thelounge/thelounge/compare/v3.2.0...v3.3.0) and [milestone](https://github.com/thelounge/thelounge/milestone/33?closed=1).
+
+This is a minor release aimed at mostly fixing bugs and increasing stability.
+
+**This release bumps required Node.js version to at least 10.16.3 or higher.**
+We suggest updating to Node.js v12, which is the latest LTS version.
+
+Feature wise, there are a couple of things that have been added. Reconnecting to a network and
```

Comment:
Suggestion: `If you reconnect to a network and the nick you want is in use, The Lounge will now attempt to regain your original nick if it sees that nick quit the network.`